wheel/src/api.rs

```diff
@@ -43,6 +61,7 @@ pub fn run_serialized_chia_program(
 fn clvm_rs(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_function(wrap_pyfunction!(run_serialized_chia_program, m)?)?;
     m.add_function(wrap_pyfunction!(serialized_length, m)?)?;
+    m.add_function(wrap_pyfunction!(skip_clvm_object, m)?)?;
```

Comment:
don't forget to add this function to `wheel/generate_type_stubs.py` and rerun it as well

Reply:
I think that's only in `chia_rs`. The python API here is small enough to manage by hand.

Reply:
oh, right